package.json

```diff
@@ -57,7 +57,7 @@
 		"arrowParens": "avoid"
 	},
 	"lint-staged": {
-		"packages/{src,test}/**/*.js": [
+		"packages/**/{src,test}/**/*.js": [
```

Comment:
Let's move this change to another PR as it doesn't seem to be necessary for route parameters